src/app/dashboard/emsZones/objects/connections.tsx

```
@@ -0,0 +1,1067 @@
+		if (!fromNode || !toNode) {
+			console.error("Connection is invalid", conn);
+			alert("Connection is invalid");
+		}
+	});
+
+	//console.log(data);
+
+	//look thought data and check against connections
+
+	for (const key in data) {
+		if (data.hasOwnProperty(key)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)